packages/docusaurus-theme-classic/src/theme/ThemedImage/styles.module.css

```diff
@@ -3,9 +3,9 @@
 }
 
 html[data-theme='light'] .themedImage--light {
-  display: block;
+  display: inherit;
 }
 
 html[data-theme='dark'] .themedImage--dark {
-  display: block;
+  display: inherit;
```

Comment:
wondering why inherit here instead of inline or initial?

it will likely does not have any impact, just wondering